mosip/aws/README.md

```diff
@@ -59,8 +59,14 @@ kubectl apply -f sc.yaml
 ```
 * If the PV gets deleted (say cluster was retarted), then you will have to define a PV connecting to this instance of storage (you will need volume ID etc). TODO: how to do this?
 
-### LongHorn
-Install [LongHorn](../longhorn/README.md).
+## The following storage classes can be used:
```

Comment:
this is aw specific section.. here we should have gp2 or gp3 or EBS or EFS mentioned.